requirements/requirements.in

```diff
@@ -8,7 +8,7 @@ boto3
 botocore
 channels
 channels-redis
-cryptography>=41.0.7  # CVE-2023-49083
+cryptography<42.0.0  # next version requires OPENSSL 3
```

Comment:
To clarify, it's not that 42+ requires OpenSSL v3, but some investigation is needed for 42+ to work with both OpenSSL v3.0.x (RHEL 9.4) and v3.2.x (RHEL 9.5).

Reply:
I'm going to paste this in the comment, mostly as you word it, "investigation is needed for 42+ to work with both OpenSSL v3.0.x (RHEL 9.4) and v3.2.x (RHEL 9.5)"